### PR TITLE
handling error body

### PR DESCRIPTION
**Pull request by Samuel-Gill**

### CODE REVIEW

**Comment by UmanShahzad** [comment]
Please link the relevant issue

File: ipinfo/handler.py [hunk 1]
```diff
@@ -140,7 +141,10 @@ def getDetails(self, ip_address=None, timeout=None):
         response = requests.get(url, headers=headers, **req_opts)
         if response.status_code == 429:
             raise RequestQuotaExceededError()
-        response.raise_for_status()
+        if response.status_code >= 400:
```
Comment (UmanShahzad) [inline ipinfo/handler.py:144]: Is this the same status code range check as `raise_for_status()`?

**Review by UmanShahzad — COMMENTED** [review]
Need to add in the async handler or no?

**Comment by UmanShahzad** [comment]
https://github.com/ipinfo/python/issues/58

File: ipinfo/handler.py (hunk 1 above)
Comment (Samuel-Gill) [reply ipinfo/handler.py:144]: Yes, `raise_for_status()` will raise an exception if the response status code falls into the 4xx or 5xx range.

**Comment by Samuel-Gill** [comment]
> Need to add in the async handler or no?

Yes, it was needed there too, I added for async too now.

**Review by UmanShahzad — APPROVED** [review]

**Pull request merged**